main.nf

```diff
@@ -150,21 +154,21 @@ if (params.readPaths){
    raw_reads_2 = Channel.create()
 
    Channel
-      .fromFilePairs( params.reads )
+      .fromFilePairs( params.input )
       .separate( raw_reads, raw_reads_2 ) { a -> [tuple(a[0], a[1][0]), tuple(a[0], a[1][1])] }
 }
 
+// SPlit fastq files
+// https://www.nextflow.io/docs/latest/operator.html#splitfastq
+
 if ( params.split_fastq ){
    raw_reads_full = raw_reads.concat( raw_reads_2 )
-   raw_reads = raw_reads_full.splitFastq( by: params.splitFastq , file: true)
```

Comment:
Maybe you should specify the parameter name changes in `CHANGELOG` too.

Reply:
The parameter has not changed. It was a typo in the current version.